### PR TITLE
refactor: Implemented basic PS style guidelines for RunTests.ps1

### PR DESCRIPTION
**Pull request by ader1990**

### CODE REVIEW

File: RunTests.ps1 [hunk 1]
```diff
@@ -9,117 +9,164 @@
...
-    {
-        Set-Variable -Name VerboseCommand -Value "" -Scope Global
+        Set-Variable -Name "VerboseCommand" -Value $VerboseCommand -Scope Global
+    } else {
+        Set-Variable -Name "VerboseCommand" -Value "" -Scope Global
     }
-    #endregion
 
-    #Validate the test parameters.
+    # Note(v-advlad): why not put the params' validation before anything else?
```
Comment (ader1990) [inline RunTests.ps1]: this needs to be addressed @iamshital can you shed some light here?

File: RunTests.ps1 [hunk 2]
```diff
@@ -9,117 +9,164 @@
...
     ValidateParameters
 
+    # Note(v-advlad): Fix typo
     ValiateXMLs -ParentFolder $WorkingDirectory
```
Comment (ader1990) [inline RunTests.ps1]: this needs to be addressed @iamshital can you shed some light here?

File: RunTests.ps1 [hunk 3]
```diff
@@ -9,117 +9,164 @@
...
-    $ARMImage = $ARMImageName.Split(" ")
+    $testXMLs = Get-ChildItem -Path "$WorkingDirectory\XML\TestCases\*.xml"
+    $setupTypeXMLs = Get-ChildItem -Path "$WorkingDirectory\XML\VMConfigurations\*.xml"
+
+    # Note(v-advlad): Why are you exploding arm image params? Are there more than one?
+    # This should be done only for Azure platform
+    $armImage = $ARMImageName.Split(" ")
     $xmlFile = "$WorkingDirectory\TestConfiguration.xml"
+
+    # Note(v-advlad): This part makes no sense
```
Comment (ader1990) [inline RunTests.ps1]: this needs to be addressed @iamshital can you shed some light here?

File: RunTests.ps1 [hunk 4]
```diff
@@ -9,117 +9,164 @@
...
-    
     #region Local Variables
-    $TestXMLs = Get-ChildItem -Path "$WorkingDirectory\XML\TestCases\*.xml"
-    $SetupTypeXMLs = Get-ChildItem -Path "$WorkingDirectory\XML\VMConfigurations\*.xml"
-    $allTests = @()
-    $ARMImage = $ARMImageName.Split(" ")
+    $testXMLs = Get-ChildItem -Path "$WorkingDirectory\XML\TestCases\*.xml"
+    $setupTypeXMLs = Get-ChildItem -Path "$WorkingDirectory\XML\VMConfigurations\*.xml"
+
+    # Note(v-advlad): Why are you exploding arm image params? Are there more than one?
```
Comment (ader1990) [inline RunTests.ps1]: this needs to be addressed @iamshital can you shed some light here?

File: RunTests.ps1 [hunk 5]
```diff
@@ -136,328 +183,279 @@ try
...
 
-    #Validate all XML files in working directory.
+    # Collect all XML files from working directory
     $allTests = CollectTestCases -TestXMLs $TestXMLs
-    
+
     #region Create Test XML
     $SetupTypes = $allTests.SetupType | Sort-Object | Get-Unique
 
+    # Note(v-advlad): This part makes no sense, with tabs and all
```
Comment (ader1990) [inline RunTests.ps1]: this needs to be addressed @iamshital can you shed some light here?

File: RunTests.ps1 [hunk 6]
```diff
@@ -9,117 +9,164 @@
...
+    # Required for Hyper-V platform
+    # Required for Azure platform if ARMImageName is not provided
+    [parameter(Mandatory=$false)]
+    [string] $OsVHD,
+
+    # Required for Azure platform
+    [parameter(Mandatory=$false)]
+    [string] $TestLocation,
+    # Required for Hyper-V platform too ??
+    [parameter(Mandatory=$false)]
```
Comment (ader1990) [inline RunTests.ps1]: this needs to be addressed @iamshital can you shed some light here?

**Comment by simonxiaoss** [comment]
Please don't write review comments into the source code. For example:
 +    # Note(v-advlad): why not put the params' validation before anything else?

This must be fixed before review.

File: RunTests.ps1 (hunk 1 above)
Comment (simonxiaoss) [reply RunTests.ps1]: Don't use source code for review question. "# Note(v-advlad): why not put the params' validation before anything else?"

**Pull request closed without merging**